lib/utils/parseJson.js

```diff
@@ -0,0 +1,11 @@
+const { parse } = require('jju/lib/parse');
+
+const parseJson = (json, revivew) => {
```

Comment:
I think this is a good idea in to have a wrapper not only because of the `fs` issue, but also because in the past it happened already that we were switching the JSON parser implementations.